spec/features/dashboard/show_spec.rb

```diff
@@ -3,14 +3,42 @@
 describe 'As a User' do
   describe 'When visiting the dashboard page' do
     before :each do
-      @user = User.create(name: 'Mr. Fake', email: 'fake@email.com', uid: '7', token: 'token')
+      user_data = stub_omniauth
+      user = User.find_or_create_by(uid: user_data[:uid])
+      user.name = user_data[:info][:name]
+      user.email = user_data[:info][:email]
+      user.token = user_data[:credentials][:token]
+      user.image = user_data[:info][:image]
+      user.save
 
-      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
 
       visit dashboard_path
     end
-    it 'Has a users name on the page' do
-      expect(page).to have_content('Mr. Fake')
+
+    it 'Has a users google information on the page' do
+      expect(page).to have_content(user.name)
+      expect(page).to have_content(user.image)
```

Comment:
I think we have to use something other than `have_content` when it comes to testing for images - this'll check for the text of the link to the image, not the image itself. Maybe check for the CSS with `have_selector` or I think I've used `have_xpath` in the past